addon/templates/components/cfb-form-editor/question.hbs

```diff
@@ -154,6 +154,15 @@
         class="uk-flex uk-flex-between uk-flex-column"
       }}
 
+      <details>
```

Comment:
Not sure if you checked the [browser support](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#Browser_compatibility).

Reply:
Oh damn, very good point. I'll look for another solution.

Reply:
Fixed